packages/docs/page-config/ui-elements/layout/examples/AllSlots.vue

```diff
@@ -0,0 +1,38 @@
+<script setup lang="ts">
+  const showLeftSidebar = ref(true)
+  const showRightSidebar = ref(true)
+</script>
+
+<template>
```

Comment:
Would be nice to have something that looks all right on mobile:

<img width="428" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/ce6a23af-2fc1-435d-b8cc-8dfed2b50175">